dev_guide/builds/build_strategies.adoc

```diff
@@ -73,92 +73,6 @@ See the xref:../../creating_images/s2i.adoc#creating-images-s2i[S2I Requirements
-to a runtime-only image.
-====
-
-For compiled languages (Go, C, C++, Java, etc.) the dependencies necessary for
-compilation might increase the size of the image or introduce vulnerabilities
-that can be exploited.
-
-To avoid these problems, S2I (Source-to-Image) introduces a two-image build process
-that allows an application to be built via the normal flow in a builder image,
-but then injects the resulting application artifacts into a runtime-only image
-for execution.
-
-To offer flexibility in this process, S2I executes an `assemble-runtime`
-script inside the runtime image that allows further customization of the
-resulting runtime image.
-
-More information about this can be found in the official
-https://github.com/openshift/source-to-image/blob/master/docs/runtime_image.md[S2I
-extended builds documents].
-
-This feature is available only for the source strategy.
-
-[source,yaml]
-----
-strategy:
-  type: "Source"
-  sourceStrategy:
-    from:
-      kind: "ImageStreamTag"
-      name: "builder-image:latest"
-    runtimeImage: <1>
-      kind: "ImageStreamTag"
-      name: "runtime-image:latest"
-    runtimeArtifacts: <2>
-      - sourcePath: "/path/to/source"
-        destinationDir: "path/to/destination"
-----
-<1> The runtime image that the artifacts should be copied to.
-This is the final image that the application will run on. This image should
-contain the minimum application dependencies to run the injected content from
-the builder image.
-<2> The runtime artifacts are a mapping of artifacts produced in the builder
-image that should be injected into the runtime image.
-`sourcePath` can be the full path to a file or directory inside the builder
-image.
-`destinationDir` must be a directory inside the runtime image where the
-artifacts will be copied. This directory is relative to the specified
-*WORKDIR* inside that image.
-
-[NOTE]
-====
-In the current implementation, you cannot have incremental extended builds thus,
-the *incremental* option is not valid with `runtimeImage`.
-====
-
-If the runtime image needs authentication to be pulled across OpenShift
-projects or from another private registry, the details can be specified
-xref:../managing_images.adoc#using-image-pull-secrets[within the image pull secret configuration].
-
-[[dev-guide-testing-your-application]]
-==== Testing your Application
-
-Extended builds offer two ways of running tests against your application.
-
-The first option is to install all test dependencies and run the tests inside
-your _builder image_ since that image, in the context of extended builds, will
-not be pushed to a registry. This can be done as a part of the `assemble`
-script for the builder image.
-
-The second option is to specify a script via the
-xref:build_hooks.adoc#dev-guide-build-hooks[postcommit hook]. This is executed
-in an ephemeral container based on the runtime image, thus it is not committed
-to the image.
```

Comment:
@csrwng In https://github.com/openshift/openshift-docs/pull/3785, this "Testing your Application" section was left in, but it looks like it's a subsection of "Extended Builds" so I'm removing it?